lms/static/scripts/ui-playground/components/ErrorComponents.js

```diff
@@ -100,221 +101,115 @@ function OAuth2RedirectErrorAppExample({ errorCode = '' }) {
-          {'"Try again"'} action.
-        </p>
-        <p>
-          Information shown to the user sometimes includes static text
-          associated with the error response error code, for example, how to fix
-          a missing Canvas file.
-        </p>
-
-        <p>TODO</p>
-
-        <h3>Errors that arise when grading assignments</h3>
-
-        <p>
-          Assignment grading occurs within the <code>basic-lti-launch</code>{' '}
-          mode of the application.
-        </p>
-
-        <p>TODO</p>
-
-        <h2>Errors that arise when configuring assignments</h2>
-        <p>
-          Errors can also arise when configuring assignments, in the{' '}
-          {"application's"} <code>content-item-selection</code> mode.
-        </p>
-        <p>
-          These errors arise from our own proxy API, and also from integrations
-          with other APIs (e.g. Google Drive).
-        </p>
-        <p>TODO</p>
       </div>
 
       <Library.Pattern title="ErrorDisplay">
         <Library.Example>
           <p>
-            All of the errors described here, with rare exception, use the{' '}
-            <code>ErrorDisplay</code> component to show their core details.
-          </p>
-          <p>
-            It is intended to be used within a Modal context, and provides a{' '}
-            <code>Scrollbox</code> to scroll content if it is too tall for the
-            containing element.
+            The <code>ErrorDisplay</code> component is used to show information
+            about errors to users. It is intended to be used within a Modal
+            context, and provides a <code>Scrollbox</code> to scroll content if
+            it is too tall for the containing element.
           </p>
           <p>
-            When information about an error is displayed to a user, we show:
+            When information about an error is displayed to a user via{' '}
+            <code>ErrorDisplay</code>, we show:
           </p>
           <ul>
             <li>
               <p>
-                <b>Prepared/canned text</b> based on <b>error code</b>
-              </p>
-              <p>
-                Canned text for different kinds of errors is defined in the
-                front-end app, for example, instructions on how to rectify a
-                missing Canvas file. Which canned text gets displayed is
-                determined by an error code provided by the back end. Depending
-                on the error context, this error code may either be in the JS
-                configuration object (error {'"modes"'}) or within the response
-                body of a problematic request to our proxy API.
-              </p>
-              <p>
-                Canned text is <b>sometimes</b> shown. It depends on whether a
-                recognized error code is provided. It is not relevant to all
-                error cases.
+                Any available <b>prepared/canned text</b> based on{' '}
```

Comment:
I would suggest to have four bullet points, one for each of the section in order of appearance:

![Errors-Hypothesis-UI-playground](https://user-images.githubusercontent.com/8555781/144079830-8c20244d-edd0-4ad8-b0fc-e495f1c0f976.png)

Reply:
Done!